src/app/Application.ts

```diff
@@ -29,24 +30,26 @@ import type { DestroyOptions } from '../scene/container/destroyTypes';
  * Any plugin that's usable for Application should contain these methods.
  * @example
  * import { ApplicationPlugin } from 'pixi.js';
- * class MyPlugin implements ApplicationPlugin
- * {
- *    static init(options)
+ *
+ * const plugin: ApplicationPlugin = {
+ *    init: (options: Partial<ApplicationOptions>) =>
  *    {
- *      // do something with options
- *    }
- *    static destroy()
+ *       // handle init here, use app options if needed
+ *    },
+ *    destroy: () =>
  *    {
- *     // destruction code here
+ *       // handle destruction code here
  *    }
  * }
  * @memberof app
+ * @see {@link app.ApplicationOptions}
+ * @ignore
```

Comment:
I cleaned it up but then I searched around and found nothing uses this. The "Plugin" vs "Mixin" vs "Extension" is a little confusing.